lib/environment/background/loadScript.js

```diff
@@ -4,5 +4,5 @@ import { apiToPromise } from '../utils/api';
 import { addListener } from './messaging';
 
 addListener('loadScript', async ({ url }, { tab: { id: tabId }, frameId }) => {
-	await apiToPromise(chrome.tabs.executeScript)(tabId, { file: url, frameId, runAt: 'document_start' });
+	await apiToPromise(chrome.scripting.executeScript)(tabId, { file: url, frameId, runAt: 'document_start' });
```

Comment:
The `executeScript` function [has changed a little](https://developer.chrome.com/docs/extensions/reference/scripting/#injection-targets):
```
addListener('loadScript', async ({ url }, { tab: { id: tabId }, frameId }) => {
	await apiToPromise(chrome.scripting.executeScript)({
		target: { tabId, frameIds: [frameId] },
		files: [url],
	});
});
```

Reply:
This solves the script error, thank you!